Example/Glia Sample Integration/ViewModifiers/AvailabilityModifier.swift

```diff
@@ -0,0 +1,16 @@
+import SwiftUI
+
+struct IsAvailable: ViewModifier {
```

Comment:
I think that this name is a little bit too generic. I see that this is so that if the media type is available, then it's solid, otherwise it's partially opaque. This is generic enough that I agree with not naming it anything related to media types, but maybe something that conveys more what is being done would be good. I see that for example the button also has a modifier called button style, maybe OpacityStyle would be a better name? You are most likely more experienced in SwiftUI than I am, so maybe there's a name that sounds more SwiftUI to this modifier :)

Reply:
That is a good comment and I agree this is very generic. However, I don't think this is necessarily bad. In addition to opacity configuration, this modifier also disables the button entirely, not allowing the user to click on it. This can be used on any button anywhere by just passing in a boolean value

Reply:
As I don't think of a better name that encapsulates both functionalities, then it's fine. If no one else says anything then it should be understandable enough

Reply:
Since this is `ViewModifier` (+ context from SwiftUI) I think it is ok to leave it as is for now.